client/python/cryoet_data_portal/src/cryoet_data_portal/_models.py

```diff
@@ -106,91 +105,87 @@ def download_everything(self, dest_path: Optional[str] = None):
 
 
 class DatasetAuthor(Model):
-    """Metadata for authors of a dataset
+    """Authors of a dataset
 
     Attributes:
-        id (int): A numeric identifier for this author
         affiliation_address (str): Address of the institution an author is affiliated with.
-        affiliation_identifier (str):  A unique identifier assigned to the affiliated institution by The Research Organization Registry (ROR).
+        affiliation_identifier (str): A unique identifier assigned to the affiliated institution by The Research Organization Registry (ROR).
         affiliation_name (str): Name of the institutions an author is affiliated with. Comma separated
-        author_list_order(int): The order in which the author appears in the publication
-        corresponding_author_status (bool):Indicating whether an author is the corresponding author
-        dataset (Dataset): An object relationship with the dataset this author corresponds to
-        dataset_id (int): Numeric identifier for the dataset this author corresponds to
+        author_list_order (int): The order in which the author appears in the publication
+        corresponding_author_status (bool): Indicating whether an author is the corresponding author
+        dataset (Dataset): An object relationship
+        dataset_id (int): None
```

Comment:
This does not have a description in `schema.graphql`, so should be fixed there or its ultimate source.